src/Microsoft.PowerApps.TestEngine/TestEngineEventHandler.cs

```diff
@@ -35,6 +35,31 @@ public void EncounteredException(Exception ex)
             {
                 Console.WriteLine($"   Assertion failed: {ex.InnerException.InnerException.Message}");
             }
+            else if (ex is UserInputException)
+            {
+                switch (ex.Message)
+                {
+                    case nameof(UserInputException.errorMapping.UserInputExceptionInvalidFilePath):
+                        Console.WriteLine($"Invalid file path. For more details, check the logs.");
```

Comment:
Not sure we can address this for the pac cli release, but for next release we need to make sure that **100%** of the user-facing output is localized.

Reply:
Can you file a work item to fix this?

Reply:
Yup.